src/wpt/urlpattern-test.ts

```diff
@@ -164,4 +157,9 @@ export default {
       'Pattern: [{"pathname":"/([\\\\d&&[0-1]])"}] Inputs: [{"pathname":"/3"}]',
     ],
   },
+  'urlpattern.https.any.js': {
+    comment:
+      "We don't currently distinguish between the HTTP and HTTPS variants of tests",
```

Comment:
I'm following this comment? Why are we skipping this?

Reply:
I tried improving the wording. Let me know if it makes more sense now.